msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -204,27 +204,12 @@ microsoftTeams.executeDeepLink(/*deepLink*/);
 
 You can open a scheduling dialog from your Teams app, as shown in the following code. This is especially useful if your app helps the user complete calendar or scheduling related tasks.
```

Comment:
Can you please remove this line as well?

Reply:
Hi! The line is removed.